tests/common/dualtor/dual_tor_utils.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import pytest
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope='session')
+def tor_mux_intf(duthosts):
+    '''
+    Returns the server-facing interface on the ToR to be used for testing 
+    '''
+    dut = duthosts[0]
```

Comment:
The comment above mentions "the ToR to be used for testing". But, here always 0th indexed ToR is used.
Do you want to use a new fixture which tells which is the ToR that is being used for a test (returns the index of it), and then use that duthost instead of hardcoding to 0?

Reply:
From my understanding, the same interfaces on each ToR should be connected to the same PTF/server ports, so using either the first or second DUT here should not make a difference.

Reply:
Checkout this `rand_one_dut_hostname` fixture in conftest. This is what we use when using first or second DUT does not make difference ( to simply avoid hardcoding and allow randomizing).
An example will be https://github.com/Azure/sonic-mgmt/blob/master/tests/bgp/test_bgp_speaker.py#L254

Reply:
As discussed offline, will add option to randomize ports in future PR